pyvodb/cli/videometadata.py

```diff
@@ -0,0 +1,71 @@
+import os.path
+from collections import OrderedDict
+import yaml
+import click
+
+from slugify import slugify
+from pyvodb.cli.top import cli
+from pyvodb.cli import cliutil
+
+
+yaml.SafeDumper.add_representer(OrderedDict,
+    lambda dumper, value: dumper.represent_dict(value.items()))
+
+
+@cli.command()
+@click.argument('city')
+@click.argument('date')
+@click.argument('outpath', default=".")
```

Comment:
If its optional, it should be an option, not an argument. Also, Click has some [special logic for paths](http://click.pocoo.org/6/api/#click.Path) that you can use.

```python
@click.option('-o', '--outpath', default=".",
              type=click.Path(dir_okay=True, file_okay=False))
```

Reply:
This is copied from `show` command so maybe fix should go there as well. In this case of metadata export, those arguments should be compulsory – at least I didn't expect somebody would like to generate folders for more than one Pyvo meeting.

Reply:
I was talking about the `outpath`, sorry. The city & date are fine!

Reply:
Oh, now I see!